src/recap

```diff
@@ -640,90 +569,85 @@ check_disk_space() {
 
 # check to see where the configuration file is located.
 # if the file is not at /etc/recap, make some noise to alert users
-if [ -f /etc/sysconfig/recap ] && [ -f /etc/recap ]
+if [ -r /etc/sysconfig/recap ] && [ -r /etc/recap ]
 then
 	echo "Configuration files exist at old (/etc/sysconfig/recap) and new locations (/etc/recap). The file from the old location will be read. Please consolidate your configuration details into /etc/recap."
-	. /etc/sysconfig/recap
-elif [ -f /etc/sysconfig/recap ] && [ ! -f /etc/recap ]
+	source /etc/sysconfig/recap
+elif [ -r /etc/sysconfig/recap ] && [ ! -r /etc/recap ]
 then
 	echo "Configuration file exists at old location (/etc/sysconfig/recap). The file will be read. Please move your configuration file to /etc/recap."
-	. /etc/sysconfig/recap
-elif [ ! -f /etc/recap ]
+	source /etc/sysconfig/recap
+elif [ ! -r /etc/recap ]
 then
 	echo "No configuration file found. Proceeding with defaults."
 else
-	. /etc/recap
+	source /etc/recap
 fi
 
 # Quit if load is higher than MAXLOAD
-LOAD=$(uptime | awk -F'load average:' '{ print $2 }' | cut -d" " -f2 | cut -d"," -f1)
-QUIT=$(echo $LOAD'>'$MAXLOAD | bc -l)
-if [ $QUIT -eq 1 ]
-then
-    echo "QUIT (LOAD:"$LOAD", MAXLOAD:"$MAXLOAD")"
-    exit 1
+LOAD=$( grep -Po '^[\d.]+' /proc/loadavg )
```

Comment:
This does actually seem to work, but I'm surprised at that.  "." is supposed to match 1 of any character, so I'd expect this regex to match the whole file, not just the first loadavg entry.  Demo:
```
galapagos ~ # grep -Po '.+' /proc/loadavg
1.46 1.40 1.13 1/854 2847
galapagos ~ #
```
The regex as it's currently written does actually work how we want, but it seems to violate what I know about regex.  I'd feel more comfortable with this if we escaped the '.' - please do?